docs/lang/articles/contribution/dev_install.md

```diff
@@ -406,18 +435,37 @@ The `develop` command serves the developers' needs better because edits to the P
 
 <TabItem value="windows">
 
-1. Run powershell as administrator and `Set-ExecutionPolicy -ExecutionPolicy UNRESTRICTED`.
-2. Run [this script](https://github.com/taichi-dev/taichi_assets/releases/download/windows_scripts/install_deps.ps1) to install all necessary dependencies:
+1. Set-up the environment variable `TAICHI_CMAKE_ARGS` with value `-DCLANG_EXECUTABLE=<Path to Clang 10>;/bin/clang.exe -DLLVM_AS_EXECUTABLE=<Path to LLVM 10>/bin/llvm-as.exe`
+2. Open the "x64 Native Tools Command Prompt" for VS2019 or VS2022. Please make sure you opened the x64 version. (Or load the Visual Studio environment yourself)
+3. Clone the Taichi repo *recursively* & install python dependencies
 
-   ```shell
-   .\install_deps.ps1
-   ```
-3. Reboot your computer.
-4. Run powershell as administrator and run [this script](https://github.com/taichi-dev/taichi_assets/releases/download/windows_scripts/build_taichi.ps1) to build Taichi from souce:
 
-   ```shell
-   .\build_taichi.ps1
-   ```
+```shell
+git clone --recursive https://github.com/taichi-dev/taichi
+
+cd taichi
+
+python -m pip install --user -r requirements_dev.txt
+```
+
+4. Build taichi by using `python setup.py develop`
+
+:::note
```

Comment:
Did we just add this part (or move from other places)? I somehow couldn't find where it was moved from hmmm

Reply:
Previously we asked the user to run `build_taichi.ps1`. Now we are asking them to build it through command line, the build script is for CI, not really for end user